### PR TITLE
Some simplifications in Docs/Simulations For GPUs

### PR DESCRIPTION
**Pull request by navidcy**

### CODE REVIEW

File: docs/src/simulation_tips.md [hunk 1]
```diff
@@ -86,12 +86,13 @@ fixes the issue by indicating to the compiler that `T₀` will not change.
 Note that the _literal_ `2π / 86400` is not an issue -- it's only the
 _variable_ `T₀` that must be declared `const`.
 
-Alternatively, passing the variable as a parameter to `GradientBoundaryCondition` also works:
+Alternatively, passing the variable as a parameter to `GradientBoundaryCondition`. To do that
```
Comment (tomchor) [inline docs/src/simulation_tips.md:89]:
```suggestion
Alternatively, we can pass the variable as a parameter to `GradientBoundaryCondition`. To do that
```

File: docs/src/simulation_tips.md [hunk 2]
```diff
@@ -188,8 +189,7 @@ For large simulations on the GPU, careful management of memory allocation may be
   ```
 
 - Try to use higher-order advection schemes. In general when you use a higher-order scheme you need
-  fewer grid points to achieve the same accuracy that you would with a lower-order one. Oceananigans
-  provides two high-order advection schemes: 5th-order WENO method (WENO) and 3rd-order upwind.
+  fewer grid points to achieve the same accuracy that you would with a lower-order one.
```
Comment (tomchor) [inline docs/src/simulation_tips.md:192]: Do we have a page discussing the advection schemes that are available? If so, we should probably link it here

Comment (navidcy) [reply docs/src/simulation_tips.md:192]: I don’t think we do. There is this —> https://clima.github.io/OceananigansDocumentation/stable/appendix/library/#Advection

Comment (navidcy) [reply docs/src/simulation_tips.md]: I added a reference to the above. It's better than nothing atm.

**Review by tomchor — APPROVED** [review]

**Pull request merged**